src/microsoft-graph-sdk-beta/Assignment.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Graph.Beta;
+using Microsoft.Graph.Beta.Models;
+
+namespace microsoft_graph_sdk
+{
+    public class Assignment
```

Comment:
class level comment header across the board with meaningful detail/description on the purpose/intent of the class